src/examples/install_specific_version.yml

```diff
@@ -1,24 +1,23 @@
 description: >
-  Install a specific version of the SecretHub CLI and inject secrets into your app.
+  Install a specific version of the SecretHub CLI.
 
 usage:
   version: 2.1
 
   orbs:
-    secrethub: secrethub/cli
+    secrethub: secrethub/cli@x.y.z
```

Comment:
Should it be possible to run the example? i.e. use a real version here.

Reply:
I took this from: https://circleci.com/orbs/registry/orb/circleci/orb-tools